use/events.js

```diff
@@ -163,71 +163,69 @@ export const useEvents = () => {
     {
       name: 'name',
       labelPosition: 'top',
-      placeholder: 'Event Name',
+      placeholder: t('events.edit.form.eventName'),
     },
     {
       name: 'description',
       labelPosition: 'top',
       component: 'TInputTextarea',
-      placeholder: 'Details (markdown)',
-      tips:
-        'Pitch yourself: Who are you? What do you offer? What do you want?\n\nTips for effective pitch:\n- Uncomplicated: It should be catchy and roll off the tongue\n- Concise: It shouldn’t take more than a minute to say or read\n- Unique: It reflects your skills, goals, and desires\n- Storyline: It covers who you are, what you offer, and where you want to be\n- Appealing: Your elevator pitch is essentially a persuasive sales pitch; the emphasis should be on what you offer',
-      description:
-        'Use [widgets](https://wedance.vip/markdown), including images and videos',
+      placeholder: t('profile.story.placeholder'),
+      tips: t('profile.story.tips'),
+      description: t('events.edit.form.description'),
     },
     {
       name: 'startDate',
       type: 'datetime-local',
-      label: 'When?',
+      label: t('events.edit.form.when'),
       labelPosition: 'top',
       simple: true,
       onChange: updateEndDate,
     },
     {
       name: 'venue',
-      label: 'Where?',
+      label: t('events.edit.form.where'),
       labelPosition: 'top',
       component: 'TInputVenue',
       simple: true,
     },
     {
       name: 'duration',
-      label: 'How long?',
+      label: t('events.edit.form.duration'),
```

Comment:
Let's simplify all the fields keys to `<object><.field>[.attribute]`
For example this one would be:

```suggestion
      label: t('event.duration'),
```